### PR TITLE
Move some crowbar cli calls to new crowbarctl

### PR DESCRIPTION
**Pull request by rsalevsky**
This change moves some crowbar cli calls to crowbarctl to test the new
cli more actively.

### CODE REVIEW

**Comment by jsuchome** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 1]
```diff
@@ -3028,7 +3028,13 @@ function update_one_proposal()
     # hook for changing proposals:
     custom_configuration $proposal $proposaltypemapped
 
-    crowbar "$proposal" proposal commit $proposaltype
+
+    if iscloudver 6plus; then
+        safely crowbarctl proposal commit $proposal $proposaltype
+    else
+        crowbar "$proposal" proposal commit $proposaltype
```
Comment (nicolasbock) [inline scripts/qa_crowbarsetup.sh]: Now that we re-arranging code, should we get rid of the quotes around `$proposal` as you did in the previous line?

Comment (dirkmueller) [inline scripts/qa_crowbarsetup.sh]: why no safely here?

**Comment by toabctl** [comment]
@rsalevsky : Failing with
```
10:51:59 +(qa_crowbarsetup.sh:3001) update_one_proposal(): safely crowbarctl proposal commit swift default
10:51:59 +(qa_crowbarsetup.sh:100) safely(): crowbarctl proposal commit swift default
10:54:59 /usr/lib64/ruby/2.1.0/net/protocol.rb:158:in `rescue in rbuf_fill': Net::ReadTimeout (Net::ReadTimeout)
```

**Comment by rsalevsky** [comment]
@toabctl Yep, there are some bugs in the crowbar-client.

**Comment by rsalevsky** [comment]
crowbarctl is fixed and the gating is green

**Comment by toabctl** [comment]
+1

**Comment by nicolasbock** [comment]
+1

File: scripts/qa_crowbarsetup.sh [hunk 2]
```diff
@@ -744,7 +744,11 @@ function get_disk_id_by_serial_and_libvirt_type()
 
 function get_all_nodes()
 {
-    crowbar machines list | LC_ALL=C sort
+    if iscloudver 6plus; then
+        safely crowbarctl node list --no-meta --plain| LC_ALL=C sort
```
Comment (vuntz) [inline scripts/qa_crowbarsetup.sh]: For a later cleanup PR: missing a space before `|`

**Comment by vuntz** [comment]
> crowbarctl is fixed

Hrm, is it also fixed in Cloud 6? For GM6+up, for instance?

**Comment by rsalevsky** [comment]
Is a shared rubygem so it is also in D:C:6 but it's not submitted as an update yet.

**Comment by vuntz** [comment]
> Is a shared rubygem so it is also in D:C:6 but it's not submitted as an update yet.

That's an issue; that probably means mkcloud is broken for QAM at the moment :/

**Comment by rsalevsky** [comment]
I will check with them and in case this is a problem I will revert or fix the changes.

**Pull request merged**